src/PlayFabBuddy.Lib/Entities/Accounts/PlayedTitlesListEntity.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace PlayFabBuddy.Lib.Entities.Accounts;
+
+/// <summary>
+/// Represents a list of Tiles an Account has played in
+/// </summary>
+public class PlayedTitlesListEntity
```

Comment:
should this be associated to a concrete player account entity or is this really indipendent?

Reply:
Unfortunately, this is independent ^^

Reply:
Let's fix this in #56